ChatAPI.Domain/Validations/Attributes/PhoneAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace ChatAPI.Domain.Validations.Attributes;
+public class PhoneAttribute : ValidationAttribute
+{
+	private static readonly string pattern = @"^\+\d+$";
+
+	protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+	{
+		if (value is null)
+			return new ValidationResult("The value cannot be null.");
+
+		string phoneNumber = value.ToString()!;
+
+		return IsValidPhoneNumber(phoneNumber)
+			? ValidationResult.Success 
+			: new ValidationResult("The phone number is not valid.");
+	}
+
+	private static bool IsValidPhoneNumber(string phone) =>
+		Regex.IsMatch(phone, pattern);
```

Comment:
Can we use the already crated numbers only attribute and just add the "+" sign in front?

Reply:
no.